deploy/roles/db/meta/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+dependencies:
+  - db_volume
+  - role: geerlingguy.ansible-role-postgresql
+    postgresql_data_dir: /mnt/data/db
+    postgresql_databases:
+    - name: tracker
```

Comment:
Could we change this to mindleaps_tracker? Total nitpick but I like to be more specific.